slick/src/test/java/docs/javadsl/DocSnippetFlow.java

```diff
@@ -31,7 +32,7 @@ public static void main(String[] args) throws Exception {
     final ActorSystem system = ActorSystem.create();
 
     // #flow-example
-    final SlickSession session = SlickSession.forConfig("slick-h2");
+    final SlickSession session = SlickSession$.MODULE$.forConfig("slick-h2");
```

Comment:
this is not needed and is less readable - I tested that the existing code compiles

```
sbt slick/test
```

I deliberately changed the Java code to add an unexpected character and the sbt command above failed - meaning that that command does compile the Java tests.

Reply:
My idea will report an error. 
![image](https://github.com/apache/incubator-pekko-connectors/assets/35491928/0b23f65c-b3d6-4c93-83e3-2798165751d8)

It maybe a bug for idea.In fact, I also test passed locally, then I wan to know how ci work with it, so I open this PR,because of lacking of github action.
I will report this for idea-scala-plugin.

Reply:
I understand it,it truely a bug for idea, because the bytecode is changed.
![image](https://github.com/apache/incubator-pekko-connectors/assets/35491928/91cf689b-9b9f-419d-a47a-588f198430a3)

Reply:
GitHub CI only runs automatically for members of the Apache GitHub group. You will have test this stuff on your laptop.

IDEA has lots of issues with compiling Scala. It gets even worse if you want to compile Scala 3.

Reply:
```scala
class TestObject

class TestHello {
  def hello() = {
    println("ok")
  }
}

object TestObject extends TestHello
```
![image](https://github.com/apache/incubator-pekko-connectors/assets/35491928/c3cfcc4f-1b36-44ea-bbde-ee062bbac705)

I write a min code of this.
Idea will not prompt  methods of object which is extended.